vertx-sql-client/src/test/java/io/vertx/sqlclient/benchmarks/AccumulatorWriteBenchmark.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2011-2022 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ */
+
+package io.vertx.sqlclient.benchmarks;
+
+import io.vertx.sqlclient.impl.accumulator.Accumulator;
+import org.openjdk.jmh.annotations.*;
+import org.openjdk.jmh.infra.Blackhole;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+import static io.vertx.sqlclient.benchmarks.Utils.generateStrings;
+
+@Threads(1)
+@State(Scope.Thread)
+@BenchmarkMode(Mode.Throughput)
+@OutputTimeUnit(TimeUnit.MILLISECONDS)
+@Warmup(iterations = 20, time = 1, timeUnit = TimeUnit.SECONDS)
+@Measurement(iterations = 10, time = 2, timeUnit = TimeUnit.SECONDS)
+@Fork(value = 1, jvmArgs = {"-Xms8g", "-Xmx8g", "-Xmn7g"})
```

Comment:
value >= 2

Reply:
Sorry, can you explain why it is needed?

Reply:
I found this https://stackoverflow.com/a/35147232/2133695

In 97039b66 I've improved the benchmarks. I'll run them tonight because the ETA with 3 forks is more than 7 hours ahead :sweat_smile:

Reply:
I've rolled-back to `ArrayListRowAccumulator` in edb479f

Latest test results do not demonstrate good results for `ChunkedRowAccumulator`.
Until we sort this out, let's continue with `ArrayListRowAccumulator`.

The biggest improvement here anyway is to avoid creating an accumulator if the database returns no rows or a single (which happens very often).